marklogic-sesame-performance/build.gradle

```diff
@@ -29,7 +30,7 @@ repositories {
 }
 
 dependencies {
-    compile('com.marklogic:marklogic-sesame:1.0.2.1-SNAPSHOT')
+    compile('com.marklogic:marklogic-sesame:2.0.0')
```

Comment:
Note to implementors that we have a new package name and will make a new version for the library that incorporates this work.